src/main/java/com/example/glovo/repository/OrderRepository.java

```diff
@@ -0,0 +1,64 @@
+package com.example.glovo.repository;
+
+import com.example.glovo.dto.OrderDto;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class OrderRepository {
+
+    private List<OrderDto> orders = List.of(
+            OrderDto.builder().products(List.of("Bananas", "Bottle of water")).customerId(12).build(),
+            OrderDto.builder().products(List.of("Nuts", "Milk", "Bread")).customerId(25).build());
+
+    public OrderDto findById(int id) {
+        return orders.get(id);
```

Comment:
follow by this logic the index in the list identifies the order id? such approuce may cause problems. e.g. there are orders with ids 1, 2, 3. second order is removed. which id should we use to get 3d order? for correct implementation, each order must have its own identifier

Reply:
make my MOCK repository using hashmap, where key is id of order

Reply:
basically the idea is to provide each order unique id using field OrderDto.id
without this approach the user can't get full information about Order (means if there is no id in the response, how the user can be sure he got appropriate response?)